### PR TITLE
feat: add PhaseTable

### PR DESCRIPTION
**Pull request by mariuswinger**
## Why is this pull request needed?

Want to have a table to show results of phases.

## What does this pull request change?

Adds that table.

## Issues related to this change:
#62

### CODE REVIEW

File: web/src/components/feature/PhaseTable.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    [`Mercury Flow (g/d)`]: [],
+  }
+  const phasesInResult: string[] = Object.keys(multiFlashResponse.phaseValues)
+  phasesInResult.forEach((phase, index) => {
+    tableInput['Phases'].push(phase)
+    tableInput['Mass Concentration'].push(
+      multiFlashResponse.phaseValues[phase]['mercury'].toString()
+    )
+    tableInput['Mole Concentration'].push(
+      multiFlashResponse.componentFractions['5'][index].toString()
```
Comment (soofstad) [inline web/src/components/feature/PhaseTable.tsx:21]: I assume the '5' is the id for mercury.
But coming back after 2 years, that would not be obvious. A small comment would help.

File: web/src/components/feature/PhaseTable.tsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    )
+    // TODO: add support for feed in kg/d
+    const hgPhaseFlow =
+      cubicFeedFlow *
+      42.29256 *
```
Comment (soofstad) [inline web/src/components/feature/PhaseTable.tsx:26]: Do we have an explanation for these constants?
Can that be added as a comment?

**Review by soofstad — CHANGES_REQUESTED** [review]
Good stuff!

Would like to see a test of the PhaseTable component.
Can pretty much just copy the MoleTable.test.tsx

File: web/src/components/feature/PhaseTable.tsx (hunk 2 above)
Comment (mariuswinger) [reply web/src/components/feature/PhaseTable.tsx:26]: No explanation. I'm guessing it is interpolated from empirical data.

**Review by soofstad — APPROVED** [review]
Good. If you rebase first, you will get a type error in the MockData in the test

**Pull request merged**